src/main/java/com/code_intelligence/jazzer/Jazzer.java

```diff
@@ -238,6 +287,26 @@ private static String appendWithPathListSeparator(String name, String... options
     return currentValue + File.pathSeparator + additionalOptions;
   }
 
+  private static Path findLibrary(List<String> candidateNames) {
+    if (isAndroid()) {
```

Comment:
Flipping the condition makes it so that the simple common case is handled first.